packages/client/wallets/aa/front/components/GetNFTsComponent.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import styles from '../styles/index.module.css';
+import { EVMAAWallet } from "@crossmint/client-sdk-aa";
+import { signMessage, signTypedData, verifyMessage } from '@/services/walletService';
+
+interface GetNFTsComponentProps {
+    aaWallet: EVMAAWallet | undefined;
+}
```

Comment:
Overall this is ok, however from this line in the index file:
```
{aaWallet && <GetNFTsComponent aaWallet={aaWallet} />}
```

I see that aaWallet cannot be undefined.